### DISCUSSION TITLE
Recursos para el aprendizaje de HTML, JavaScript y otros lenguajes

### DISCUSSION THREAD

**Discussion by irodzu** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#102] [General]
Buen día compañeros,

Les comparto los siguientes links con recursos para nuestro aprendizaje.

Esta es una página de referencia que contiene todos los elementos utilizados en HTML, JavaScript, Python y otros lenguajes de programación. Pueden encontrar ejercicios y ejemplos de uso de cada elemento para que aprendan a ponerlo en práctica.

[W3Schools](https://www.w3schools.com/html/default.asp)

![image](https://user-images.githubusercontent.com/25829182/155447446-637b5118-6ce5-4db8-8124-bd30bb3bdbb5.png)

Este es el libro _**Eloquent JavaScript**_ que nos ha mencionado el MC, lo pueden consultar de manera gratuita sin infringir derechos de autor o copyright.

[Eloquent JavaScript](https://eloquentjavascript.net/)

![image](https://user-images.githubusercontent.com/25829182/155447500-040bdda6-6629-4ef9-9c8a-456a09e080fb.png)

Les comparto también este editor de código llamado Brackets, es muy ligero, fácil de usar y limpio en su estética. Está enfocado hacia el desarrollo Web y lenguajes como HTML, CSS y JavaScript, aunque también soporta otros lenguajes. Personalmente es el que uso para desarrollo Web desde hace un par de años.

[Brackets](https://brackets.io/)

![image](https://user-images.githubusercontent.com/25829182/155447544-bbc6d978-6e91-41f9-9a0e-0ce253ee5d76.png)

De momento es todo, espero les sean útiles en su camino de explorers. Saludos

**Comment by JuanSalvador73** [comment]
Excelente aporte, gracias. También uso Brackets, desafortunadamente ya retiraron soporte de ese software y los plugins - como los de HTML - hay que bajarlos de su sitio, ya no te deja hacerlo online. Saludos :D

**Comment by SandroBorrallez** [comment]
Muchas gracias por tomarte el tiempo de compartir esta aportación.
